agent/util-scripts/gold/pbench-metadata-log/test-22.txt

```diff
@@ -46,7 +46,6 @@ iter3
 /var/tmp/pbench-test-utils/pbench/metadata.log:config = 
 /var/tmp/pbench-test-utils/pbench/metadata.log:date = 1900-01-01T00:00:00
 /var/tmp/pbench-test-utils/pbench/metadata.log:rpm-version = 
-/var/tmp/pbench-test-utils/pbench/metadata.log:iterations = iter1, iter2, iter3
```

Comment:
Yes, `pbench-metadata-log` can't find the `.iteration` file because it is still located in the `tmp/` directory.  If we just move `samples/pbench-metadata-log/test-22/pbench/tmp/pbench.iterations` to `samples/pbench-metadata-log/test-22/pbench/.iterations`, the unit test should pass without the removal of line 49.

Reply:
This is still not right, it appears.  Line 49 in the original is being removed.  It should stay.  It appears the `.iterations` file has been moved to the wrong location.  See the  comment above for the correct place.